src/components/registration/steps.jsx

```diff
@@ -33,29 +33,20 @@ require('./steps.scss');
 const DEFAULT_COUNTRY = 'us';
```

Comment:
note that now, the main function of DEFAULT_COUNTRY has moved into the country-data library itself

Reply:
Do you mean because the library supplies a list with several "default" countries at the top?

Reply:
Yes. So we no longer use DEFAULT_COUNTRY to sort a country to the top in this file.